internal/app/proxyd/internal/frontend/frontend.go

```diff
@@ -423,3 +351,20 @@ func (r *ReverseProxy) bootstrap(ctx context.Context) {
 		}(ctx, idx, endpoint)
 	}
 }
+
+// Backends returns back a copy of the current backends known to proxyd
+func (r *ReverseProxy) Backends() map[string]*backend.Backend {
+	r.mux.Lock()
+	defer r.mux.Unlock()
+	backends := make(map[string]*backend.Backend)
+	for name, be := range r.backends {
```

Comment:
This threw me off at first glance. I'd like to use `uid` and `addr` to be consistent with the rest of the code.